themes/default/content/docs/intro/concepts/plans.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Plans"
+meta_desc: Learn about Pulumi update plans and how they can be used.
+menu:
+  intro:
+    parent: concepts
+    weight: 10
+
+aliases:
+  - /updateplans/
+---
+
+{{% notes type="warning" %}}
+Update plans are currently in experimental preview and will only show up in `--help` if the environment variable `PULUMI_EXPERIMENTAL` is set to `true`.
+{{% /notes %}}
+
+The result of previews can be saved to an _update plan_, this can then be used during a later update to constrain the update only to the operations that were planned.
```

Comment:
Can we throw a sentence or two in here for reasons why people would want to use this feature - scenarios where this would be useful in practice? e.g. "Constraining an update to a plan may be useful in organizations with strict approval processes."